cnames_active.js

```diff
@@ -838,6 +838,7 @@ var cnames_active = {
   "hookstate": "hookstate.netlify.com", // noCF
   "hooloo": "hooloo.github.io", // noCF? (don´t add this in a new PR)
   "how-to-mithril": "stephanhoyer.github.io/how-to-mithril",
+  "hpu": "world.codeforces.site/keyboard/",
```

Comment:
```suggestion
  "hpu": "world.codeforces.site",
```
CNAME'ing to a path isn't possible, do you want this to CNAME to `world.codeforces.site`?